views/campground_requests.py

```diff
@@ -0,0 +1,63 @@
+import sqlite3
+from models import Campground
+
+def get_all_campgrounds():
+    """Gets all campgrounds
+    Returns:
+        dict: All campground dictionaries"""
+    with sqlite3.connect("./national_park.sqlite3") as conn:
```

Comment:
I notice that the string of "./national_park.sqlite3` is repeated many times in this module. I'm curious if you think it would be a good idea to store that string in a variable at the top of the module...

```py
DB_NAME = "./national_park.sqlite3"
```

And then use that in all of the connections.

```py
with sqlite3.connect(DB_NAME) as conn:
```